jetstream.md

```diff
@@ -0,0 +1,136 @@
+
+![](jetstream/images/login9.png)
+
+- **Navigate back to 'Projects' and click on your new instance's name to see more information related to the instance you just created! and Copy the IP address of your instance created.**
+
+![](jetstream/images/login10.png)
+
+> **Great! We have now built our very own remote virtual machine with all the software pre-installed. Next we will use SSH-Secure-Login to access these remote instances from our laptop !!!.**
+
+# SSH Secure-Login
+
+![](jetstream/images/ssh.png)
+
+- macOS & LINUX users can open a Terminal window now.
+- Windows 10 users with ubuntu can open a terminal 
+- If your windows doesn't come with ubuntu distribution, users start a new session in git bash
+	+ Start a new session; Fill in your "remote host" the IP address of your virtual machine; select "specify username" and enter your Jetstream username; Click OK.
+
+
+- **Establish a secure-login to the instance by typing the following:**
+
+```
+$ ssh your_Jetstreamusername@ip_address
+```
+
+![](jetstream/images/login11.png)
+
+- **This should log you into Jetstream and you should see a screen like this; Enter 'yes' and then enter your Jetstream password.**
+
+> Your cursor will not move or indicate you are typing as you enter your password. If you make a mistake, hit enter and you will be prompted again.
+
+![](jetstream/images/login12.png)
+
+> **Success !!! We have established connections with our instances. Proceed to the Tutorial section.**
+
+# Instance Maintenance
+
+> To end your current session on an Instance and close SSH connection, type 'exit'
+
+#### Jetstream Dashboard
+
+![](jetstream/images/jet_dashboard.png)
+
+### Instance Actions
+
+![](jetstream/images/jet_actions.png)
+
+| Action | Description |
+| ----- | ----- |
+| Report | instance exhibiting unexpected behavior? Report here |
+| Image | Request an image (a type of template for a virtual machine) of a running instance |
+| Suspend | Suspending an instance frees up resources for other users and allows you to safely preserve the state of your instance without imaging. Your time allocation no longer counts against you in the suspended mode |
+| Shelve | Shelving will safely preserve the state of your instance for later use. And, it frees up resources for other users . In fact, it is the best way to reduce resource usage when compared with other actions, like "suspend" and "stop".Your time allocation no longer counts against you in the shelved mode |
+| Stop | Stop an instance |
+| Reboot | Reboot an instance |
+| Redeploy | Redeploying an instance will allow you to fix instances that show up as 'active - deploy_error' |
+| Delete | following instance will be shut down and all data will be permanently lost |
+
+> **Note: It is advisable to delete the machine if you are not planning to use it in future to save valuable resources. However if you want to use it in future, you can suspend it. Notice: IP address changes**
```

Comment:
stop and suspend each use like 25 or 50% of the resources as a running instance. you need to shelve an instance if you want to not use any SUs.